src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests.cs

```diff
@@ -5470,13 +5470,33 @@ class Goo<T>
 }
 ";
 
-            var compilation = CreateCompilation(source);
+            var compilation = CreateCompilation(source, parseOptions: CSharpParseOptions.Default.WithLanguageVersion(LanguageVersion.CSharp7));
```

Comment:
`(2,2): error CS0404: Cannot apply attribute class 'Goo<T>' because it is generic` below

Feels like we should report a LangVer error her instead (updating the LangVer fixes this error)